src/code/z_view.c

```diff
@@ -438,7 +462,10 @@ s32 func_800AB2C4(View* view) {
     return 1;
 }
 
-s32 func_800AB560(View* view) {
+/**
+ * Apply scissor, viewport, view and projection (perspective) to OVERLAY_DISP.
+ */
+s32 View_ApplyPerspectiveToOverlay(View* view) {
```

Comment:
`View_ApplyOrthoToOverlay` says `Apply scissor, viewport and projection (ortho) to OVERLAY_DISP.`

I'm trying to figure out if the extra element in the `View_ApplyPerspectiveToOverlay` comment "`view`" is a typo or if there's actually a difference between the two ortho/persp variants?

Reply:
`view` refers to the view transform, as separate from the projection transform. The view transform is done by `guLookAt` and isn't done in the ortho version.